ads/aduptech.md

```diff
@@ -0,0 +1,81 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Ad Up Technology
+
+Please visit [www.adup-tech.com](http://www.adup-tech.com) for more information
+on how to get required ad tag or placement keys.
+
+## Examples
+
+### Fixed size
+
+```html
```

Comment:
Since we are here, is `layout=responsive` also applicable here? It's a layout that automatically resizes the element based on the available width while preserving the aspect-ratio for height.